Frontend/src/components/EditEvent/EditEvent/EditEvent.tsx

```diff
@@ -280,7 +280,7 @@ export const EditEvent = ({ eventResult: event, updateEvent }: IProps) => {
               className={style.textAreaContainer}
               label={labels.program}
               placeholder={placeholders.program}
-              value={event.program!!}
+              value={event.program!}
```

Comment:
Det her blir vel ikke _egentlig_ helt riktig? Når man trykker på "Legg til program" settes `hasProgram` til `true`, men _selve_ programmet `event.program` er jo fortsatt `undefined`. Foreslår å heller gi en fallback-verdi:

```suggestion
              value={event.program ?? ''}
```

Reply:
👍